src/w3c/style.js

```diff
@@ -81,26 +87,30 @@ function styleMover(linkURL) {
 }
 
 export function run(conf) {
-  if (!conf.specStatus) {
-    const msg = "`respecConfig.specStatus` missing. Defaulting to 'base'.";
-    conf.specStatus = "base";
-    showWarning(msg, name);
-  }
-
+  const canonicalStatus = conf.specStatus?.toUpperCase() ?? "";
   let styleFile = "W3C-";
```

Comment:
Nit:
```suggestion
  let styleFile = "";
```
And use `W3C-` prefix explicitly without `+=`.

Reply:
Yeah, the += was bugging me too... I went back and forth on that.